src/HasWishlists.php

```diff
@@ -4,9 +4,27 @@
 
 use DB;
 use Exception;
+use Illuminate\Support\Facades\Auth;
 
 trait HasWishlists
 {
+    /**
+     * Count wishlist for user.
+     *
+     * @param  array  $collections
+     * @return int
+     */
+    public function countWishlist(array $collections = []): int
+    {
+        $query = DB::table('wishlist')->where('user_id', Auth::id());
```

Comment:
Please use ```$this->id``` for the user ID for consistency; in this way we can also skip the use of the Auth facade.

Reply:
@lamalamaMark yeah, good idea 👍